ui/raidboss/data/04-sb/trial/suzaku.ts

```diff
@@ -45,23 +44,29 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'Suzaku Southron Star',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '3234', source: 'Suzaku', capture: false }),
+      netRegex: { id: '3234', source: 'Suzaku', capture: false },
       response: Responses.aoe(),
     },
     {
+      // TODO: HeadMarker trigger
       id: 'Suzaku Phantom Flurry',
-      netRegex: NetRegexes.startsUsing({ id: '3231', source: 'Suzaku' }),
+      type: 'HeadMarker',
+      // netRegex: { id: '3231', source: 'Suzaku' },
+      netRegex: { id: '3231' },
       response: Responses.tankBuster(),
     },
     {
+      // TODO: fix HeadMarker
```

Comment:
?

Reply:
I guess I may misunderstood here, doesn't head marker trigger need something like "firstHeadMarker" and calculate offset or something?

Reply:
Or these are not stale but actually working trigger 🤔

Reply:
And guessing from your reaction it should be a correct trigger. I'll remove this comment

Reply:
Only savage fights starting in e1s and ultimates (I think?) need headmarker offsets.  Everything else should be fine.  My understanding is that these should be working.

Mostly, in a conversion patch I'd expect only conversions (even if the existing trigger is broken) and if you want to change logic or fix things up or change code, then do that separately.  There's so much stuff to look through that it's easier (for me) when huge patches are only automatic conversion and nothing manual (and if there is manual change, then do that separately, first or after).

Reply:
Ok.

I generate a patch for all manually editing:

```diff
diff --git b/ui/raidboss/data/00-misc/general.ts a/ui/raidboss/data/00-misc/general.ts
index 4ff4be5a6..0b467151c 100644
--- b/ui/raidboss/data/00-misc/general.ts
+++ a/ui/raidboss/data/00-misc/general.ts
@@ -1,4 +1,3 @@
 import ZoneId from '../../../../resources/zone_id';
 import { RaidbossData } from '../../../../types/data';
 import { TriggerSet } from '../../../../types/trigger';
@@ -199,6 +198,7 @@ const triggerSet: TriggerSet<Data> = {
       // the message is sent to this channel; when a ready check is invoked by others, then it
       // would be sent to the 0239 channel.  (Sometimes this is also sent to 0139, unknown why?)
       id: 'General Ready Check',
+      type: 'GameLog',
       netRegex: { line: '(?:You have commenced a ready check|\\y{Name} has initiated a ready check).*?', code: ['0039', '0139', '0239'], capture: false },
       sound: '../../resources/sounds/Overwatch/D.Va_-_Game_on.webm',
       soundVolume: 0.6,
diff --git b/ui/raidboss/data/02-arr/raid/t13.ts a/ui/raidboss/data/02-arr/raid/t13.ts
index 98258e1cc..17f18d960 100644
--- b/ui/raidboss/data/02-arr/raid/t13.ts
+++ a/ui/raidboss/data/02-arr/raid/t13.ts
@@ -99,6 +99,7 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'T13 Earthshaker',
+      type: 'HeadMarker',
       netRegex: { id: '0028' },
       condition: Conditions.targetIsYou(),
       response: Responses.earthshaker(),
diff --git b/ui/raidboss/data/02-arr/trial/titan-ex.ts a/ui/raidboss/data/02-arr/trial/titan-ex.ts
index 3fc8c3111..d59b15f4f 100644
--- b/ui/raidboss/data/02-arr/trial/titan-ex.ts
+++ a/ui/raidboss/data/02-arr/trial/titan-ex.ts
@@ -98,6 +98,7 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'TitanEx Upheaval',
+      type: 'StartsUsing',
       // Five second cast time.
       netRegex: { source: 'Titan', id: '5BA', capture: false },
       response: Responses.knockback('info'),
diff --git b/ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.ts a/ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.ts
index b86cced37..16ed4180d 100644
--- b/ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.ts
+++ a/ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.ts
@@ -38,6 +38,7 @@ const triggerSet: TriggerSet<Data> = {
   triggers: [
     {
       id: 'Facility Petrifaction',
+      type: 'StartsUsing',
       netRegex: { id: '10EB', source: 'Harmachis', capture: false },
       response: Responses.lookAway(),
     },
@@ -53,16 +54,19 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'Facility Dark Orb',
+      type: 'StartsUsing',
       netRegex: { id: '10FC', source: ['Igeyorhm', 'Lahabrea'] },
       response: Responses.tankBuster(),
     },
     {
       id: 'Facility Shadow Flare',
+      type: 'StartsUsing',
       netRegex: { id: '1109', source: 'Ascian Prime', capture: false },
       response: Responses.aoe(),
     },
     {
       id: 'Facility Annihilation',
+      type: 'StartsUsing',
       netRegex: { id: '110A', source: 'Ascian Prime', capture: false },
       response: Responses.aoe(),
     },
diff --git b/ui/raidboss/data/03-hw/dungeon/fractal_continuum.ts a/ui/raidboss/data/03-hw/dungeon/fractal_continuum.ts
index b063ebf55..f9c6917ad 100644
--- b/ui/raidboss/data/03-hw/dungeon/fractal_continuum.ts
+++ a/ui/raidboss/data/03-hw/dungeon/fractal_continuum.ts
@@ -32,21 +32,25 @@ const triggerSet: TriggerSet<Data> = {
   triggers: [
     {
       id: 'Fractal Rapid Sever',
+      type: 'StartsUsing',
       netRegex: { id: 'F7A', source: 'Phantom Ray' },
       response: Responses.tankBuster(),
     },
     {
       id: 'Fractal Slash',
+      type: 'StartsUsing',
       netRegex: { id: 'F83', source: 'Minotaur', capture: false },
       response: Responses.awayFromFront(),
     },
     {
       id: 'Fractal Swipe',
+      type: 'StartsUsing',
       netRegex: { id: 'F81', source: 'Minotaur', capture: false },
       response: Responses.awayFromFront(),
     },
     {
       id: 'Fractal Small Swing',
+      type: 'StartsUsing',
       netRegex: { id: 'F82', source: 'Minotaur', capture: false },
       response: Responses.getOut(),
     },
@@ -85,6 +89,7 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'Fractal Alarums',
+      type: 'AddedCombatant',
       netRegex: { name: 'Clockwork Alarum', capture: false },
       suppressSeconds: 5,
       response: Responses.killAdds(),
diff --git b/ui/raidboss/data/03-hw/dungeon/xelphatol.ts a/ui/raidboss/data/03-hw/dungeon/xelphatol.ts
index 44ef6382a..480262d98 100644
--- b/ui/raidboss/data/03-hw/dungeon/xelphatol.ts
+++ a/ui/raidboss/data/03-hw/dungeon/xelphatol.ts
@@ -38,16 +38,19 @@ const triggerSet: TriggerSet<Data> = {
   triggers: [
     {
       id: 'Xelphatol Long Burst',
+      type: 'StartsUsing',
       netRegex: { id: '19C8', source: 'Nuzal Hueloc', capture: false },
       response: Responses.aoe(),
     },
     {
       id: 'Xelphatol Hot Blast',
+      type: 'StartsUsing',
       netRegex: { id: '19CC', source: 'Floating Turret', capture: false },
       response: Responses.getUnder(),
     },
     {
       id: 'Xelphatol On High',
+      type: 'StartsUsing',
       netRegex: { id: '19CF', source: 'Dotoli Ciloc', capture: false },
       response: Responses.knockback(),
     },
@@ -70,6 +73,7 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'Xelphatol Ixali Aero AOE',
+      type: 'StartsUsing',
       netRegex: { id: '19D5', source: 'Tozol Huatotl', capture: false },
       response: Responses.aoe(),
     },
diff --git b/ui/raidboss/data/04-sb/trial/suzaku.ts a/ui/raidboss/data/04-sb/trial/suzaku.ts
index f531826bd..07647ac4e 100644
--- b/ui/raidboss/data/04-sb/trial/suzaku.ts
+++ a/ui/raidboss/data/04-sb/trial/suzaku.ts
@@ -49,17 +49,20 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'Suzaku Phantom Flurry',
+      type: 'StartsUsing',
       netRegex: { id: '3231', source: 'Suzaku' },
       response: Responses.tankBuster(),
     },
     {
       id: 'Suzaku Rekindle',
+      type: 'HeadMarker',
       netRegex: { id: '3230' },
       condition: Conditions.targetIsYou(),
       response: Responses.spread(),
     },
     {
       id: 'Suzaku Ruthless Refrain',
+      type: 'StartsUsing',
       netRegex: { id: '3230', source: 'Suzaku', capture: false },
       response: Responses.knockback(),
     },

```

Reply:
And others and done my scripts, these doesn't pass the test so I edit them.